dbscripts/xml/upgrade.xml

```diff
@@ -68,6 +68,7 @@
 		<migration class="PKP\migration\upgrade\v3_4_0\I2890_AddSetNullForOnDeleteToReviewRoundIdForeign"/>
 		<migration class="APP\migration\upgrade\v3_4_0\I7901_Duplicate_OAI_IDs"/>
 		<migration class="PKP\migration\upgrade\v3_4_0\I7366_UpdateUserAPIKeySettings"/>
+		<migration class="PKP\migration\upgrade\v3_4_0\I7463_LocaleColumn"/>
```

Comment:
weirdly this migration was missing in the upgrade list and causing to fail the upgrade .

Reply:
Filed an issue related to this: https://github.com/pkp/pkp-lib/issues/10293